pkg/rancher/rancher.go

```diff
@@ -225,9 +225,14 @@ func (r *Rancher) Start(ctx context.Context) error {
 		if err := dashboarddata.Add(ctx, r.Wrangler, localClusterEnabled(r.opts), r.opts.AddLocal == "false", r.opts.Embedded); err != nil {
 			return err
 		}
-		return r.Wrangler.StartWithTransaction(ctx, func(ctx context.Context) error {
+		err := r.Wrangler.StartWithTransaction(ctx, func(ctx context.Context) error {
 			return dashboard.Register(ctx, r.Wrangler)
 		})
+		if err != nil {
+			return err
+		}
+
+		return forceUpgradeLogout(r.Wrangler.Core.ConfigMap(), r.Wrangler.Mgmt.Token(), "v2.6.0")
```

Comment:
```suggestion
		return forceUpgradeLogout(r.Wrangler.Core.ConfigMap().Cache(), r.Wrangler.Mgmt.Token().Cache(), "v2.6.0")
```

Reply:
@paynejacob , I don't see this addressed. Was it discussed with @dramich and determined the change is not needed?

Reply:
It was addressed in the body of the function.  I need read and write operations so I use the cache internally for the read operations.  If I pass the cache here I would not be able to write changes.

Reply:
Ah, I see. Thank you for the explanation.